src/NuGet.Core/NuGet.Build.Tasks.Pack/Pack.targets

```diff
@@ -139,6 +139,51 @@ Copyright (c) .NET Foundation. All rights reserved.
           TaskParameter="TargetOutputs"
           ItemName="_ProjectReferences" />
     </MSBuild>
+
+    <MSBuild
+      Condition="'$(PackBuildOutput)' == 'true'"
+      Projects="$(MSBuildProjectFullPath)"
+      Targets="BuiltProjectOutputGroup"
+      Properties="TargetFramework=%(_TargetFrameworks.Identity)
```

Comment:
Do you need a ; after this? It doesn't always seem to matter, but there are other properties on the next line